Tests/Functional/ToolsTest.php

```diff
@@ -42,6 +43,73 @@ public function testCreateAllSpool()
         global $TYPO3_CONF_VARS;
         $TYPO3_CONF_VARS['MAIL']['transport'] = 'Swift_NullTransport';
 
+        /** @var \Ecodev\Newsletter\Domain\Repository\NewsletterRepository $newsletterRepository */
```

Comment:
In this file you are basically duplicating the entire test and add a few assertions for the email. I think adding the assertions to the existing would be enough. Reducing time for testing and code duplication.